test/Rings/groebner-test.jl

```diff
@@ -1,3 +1,4 @@
+using Test: Error
```

Comment:
Why is this needed? If it isn't, please remove again.

Reply:
It is not, my language server just keeps adding in these import statements if it thinks something is out of scope... I will pay closer attention to it in the future.